cc/position.cc

```diff
@@ -51,12 +51,13 @@ std::array<inline_vector<Coord, 4>, kN* kN> kNeighborCoords = []() {
   return result;
 }();
 
-inline const inline_vector<Coord, 4>& GetNeighborCoords(Coord c) {
+}  // namespace
+
+
+inline_vector<Coord, 4>& position::GetNeighborCoords(Coord c) {
```

Comment:
It'd be nice to check the generated code to verify that the compiler still inlines the GetNeighborCoords calls:

```
bazel build -c opt --save_temps cc:position
view bazel-bin/cc/_objs/position/cc/position.pic.s
```

Reply:
Didn't know I could do this :) and it's good you mentioned to check because they are no longer inline. I'll follow up tomorrow.